packages/uniforms-bootstrap3/__tests__/DateField.tsx

```diff
@@ -43,13 +43,18 @@ test('<DateField> - renders an input with correct type', () => {
   expect(wrapper.find('input').prop('type')).toBe('datetime-local');
 });
 
-test('<DateField> - renders an input with correct disabled state', () => {
-  const element = <DateField name="x" disabled />;
-  const wrapper = mount(element, createContext({ x: { type: Date } }));
+const cases = ['datetime-local', 'date'] as DateFieldProps['type'][];
 
-  expect(wrapper.find('input')).toHaveLength(1);
-  expect(wrapper.find('input').prop('disabled')).toBe(true);
-});
+test.each(cases)(
+  '<DateField> - renders a Input with correct type : "%s"',
+  (type: DateFieldProps['type']) => {
```

Comment:
1. This can be simplified with `as const`:
    ```ts
    test.each(['date', 'datetime-local'] as const)(
      '<DateField> - renders a Input with correct type : "%s"',
      type => {
    ```
2. Revert the removal of the `renders an input with correct disabled state` test?